case_studies/los_altos_high_school/15_09_11_first_meeting/recap.md

```diff
@@ -1 +1,49 @@
 # Recap
+
+## Summary
+
+- 13 people showed up due to technical difficulties
```

Comment:
What were these technical difficulties? Is it possible for other clubs to run into these issues?